src/gluonts/evaluation/_base.py

```diff
@@ -142,8 +161,20 @@ def __call__(
             total=num_series,
             desc="Running evaluation",
         ) as it, np.errstate(invalid="ignore"):
-            for ts, forecast in it:
-                rows.append(self.get_metrics_per_ts(ts, forecast))
+            if self.num_workers > 0:
+                mp_pool = multiprocessing.Pool(
+                    initializer=_worker_init(self), processes=self.num_workers
+                )
+                rows = mp_pool.map(
+                    func=_worker_fun,
+                    iterable=iter(it),
```

Comment:
this iter(...) call is necessary for some reason, otherwise tests fail. I checked, and without it in some tests the first instance returned by next is just a string showing the progress bar?

Reply:
Since we are iterating a tqdm object, I'm not sure what the progress bar will look like when you enable multiprocessing. Maybe what you observing is related to tqdm interfering with multiprocessing?

Reply:
Yes, it's due to tqdm, however, the progress bar does work in mp.